docs/source/tutorials/faq.ipynb

```diff
@@ -271,6 +271,18 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "13228a99-5d3f-47c0-87e5-2290d16461c4",
+   "metadata": {},
+   "source": [
+    "Methods that call `filter.find_label_issues` use `label_issues_batched` internally if `low_memory=True` is passed as a parameter. The following methods provide this option - \n",
```

Comment:
Can we just move this under it's own FAQ question:

How do I get results faster?